### ISSUE TITLE
How to find the annotation file that causes the error "Wrong annotation: x = 0, y = 0, < 0 or > 1"?

### ISSUE THREAD

**Issue by monocongo** [AlexeyAB/Yolo_mark#161]
I am training YOLOv3 with a custom dataset. In the training output I infrequently see this message:
```
Wrong annotation: x = 0, y = 0, < 0 or > 1
```
My assumption is that this means that one or more of my training annotations contains bad values. However there is nothing I can see that might help me to determine which annotation file the error refers to, so finding and fixing this annotation will be like finding a needle in a haystack. 

How might I find the offending annotation file?

Thanks in advance for any suggestions or insight.

**Comment by monocongo** [comment]
I assume that this indicates that I have negative and/or greater than 1.0 values in my annotations, no?

If so then it should be simple enough to write a Python script to clean the annotations.

**Comment by monocongo** [comment]
Sorry, I opened this issue in the wrong repository, and besides, it has already been resolved in the darknet repository where I meant to open it: https://github.com/AlexeyAB/darknet/issues/3895. 

TL; DR: The file `bad_label.list` gives a list of the offending annotation files that cause this error.

**Comment by hobbiit** [comment]
I am not able to find the bad_label.list file. Can you please let me know where I can find it
